.github/workflows/build-test-push.yml

```diff
@@ -0,0 +1,46 @@
+name: Build-Test-Push
+on:
+  - push
+  - pull_request
+jobs:
+  Build-Test-Push:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Build main image
+        run: |
+          docker build --target base \
+          -t puppet-dev-tools:latest \
+          --build-arg VCS_REF=$(git rev-parse --short HEAD) \
+          --build-arg GH_USER=${{ secrets.DOCKERHUB_USERNAME }} \
```

Comment:
This is wrong... I’ll fix it. I just realized it worked in my testing because my GitHub and Docker Hub usernames are the same.